tests/conftest.py

```diff
@@ -0,0 +1,342 @@
+"""
```

Comment:
I looked at that document a bit and from what I can tell yield is needed if we are doing any kind of tear down after each fixture and we are not. As far as I know since each fixture is only scoped to function, after it gets used it gets torn down regardless. I can switch it to yield, but I don't know if that would make any difference

Reply:
yes, exactly and caching / decaching the API is one of these teardowns.
(Just hidden in the context manager).
So the API needs to be a yield fixtures, nodes. I am not so sure about.